scalalib/src/dependency/versions/ModuleDependenciesVersions.scala

```diff
@@ -3,10 +3,10 @@ package mill.scalalib.dependency.versions
 import mill.scalalib.JavaModule
 
 private[dependency] final case class ModuleDependenciesVersions(
-    module: JavaModule,
```

Comment:
What's the motivation for this change?

Reply:
The issue is that `JavaModule` is not picklable, hence it can't be used in the updates task's result

Reply:
Yeah, that makes sense.